rpc/src/rpc.rs

```diff
@@ -3698,6 +3700,7 @@ pub mod rpc_full {
             meta: Self::Metadata,
             data: String,
             config: Option<RpcSimulateTransactionConfig>,
+            account_overrides: Option<AccountOverrides>,
```

Comment:
Instead of creating another parameter, this should go into `RpcSimulateTransactionConfig`.